etc/services.d/dump1090/run

```diff
@@ -16,6 +16,11 @@ DUMP1090_CMD+=(--fix)
 ALLOW_MODEAC="$(piaware-config -show allow-modeac | tr '[:upper:]' '[:lower:]')"
 ALLOW_MLAT="$(piaware-config -show allow-mlat | tr '[:upper:]' '[:lower:]')"
 
+# Allow selection of RTLSDR device by serial number or index value
+if [ -n "${DUMP1090_DEVICE}" ]; then
+  DUMP1090_CMD+=(--device ${DUMP1090_DEVICE})
```

Comment:
Are you able to quote `${DUMP1090_DEVICE}`?

From shellcheck:
```
  DUMP1090_CMD+=(--device ${DUMP1090_DEVICE})
                          ^----------------^ SC2206: Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.
```
For more information:
  https://www.shellcheck.net/wiki/SC2206 -- Quote to prevent word splitting/g...

Reply:
Yes, I fixed it in the 2nd commit.  Did the first one too quickly and got it wrong.  Thanks!